GTBitsOfGood/national-npp
# Cullen samrat/58 nonproft create issue

## What is this?

- User Interface for Create Issue Page
- Ability to Submit an Issue
- Saving a submitted issue to the database


## Setup

- In MongoDB make sure your associated nonprofit is: `617ea5f3069ce109e2ae9f6b` (because this is the nonprofit associated to the project this issue is under)


## Steps to Test

- Go to localhost:3000/nonprofit/issues/create
- Test Submitting an issue with all information filled out
- Test submitting an issue with partial information (should not work)

## Associated Issues

#58 


## Additional Instructions

- Image upload functionality currently has not been tested--I've asked Cullen to build out the UI to support uploading images; once that gets done. we can test the upload functionality
